Wikipedia/Code/SearchViewController.swift

```diff
@@ -84,7 +84,7 @@ class SearchViewController: ArticleCollectionViewController, UISearchBarDelegate
     }
     
     var siteURL: URL? {
-        return searchLanguageBarViewController?.currentlySelectedSearchLanguage?.siteURL() ?? NSURL.wmf_URLWithDefaultSiteAndCurrentLocale()
+        return searchLanguageBarViewController?.currentlySelectedSearchLanguage?.siteURL() ?? MWKLanguageLinkController.sharedInstance().appLanguage?.siteURL()
```

Comment:
Should `NSURL.wmf_URLWithDefaultSiteAndCurrentLocale()` still be a fallback after `MWKLanguageLinkController.sharedInstance().appLanguage?.siteURL()`?

Reply:
sure